src/NUnitFramework/framework/Attributes/AttributeHierarchy.txt

```diff
@@ -7,7 +7,7 @@ NUnitAttribute
 	DataAttribute
 		ValuesAttribute               : IParameterDataSource
 			RandomAttribute           : IParameterDataSource
-			RangeAttribute
+		RangeAttribute
```

Comment:
RangeAttrivute implements `: IParameterDataSource` now - not that this file is vastly important!

Reply:
Funny thing... this is a file I created for my own use a long time ago and accidentally committed. I would have removed it, but by the time I discovered it other people had started to maintain it.